content/rdi/quickstart/write-behind-guide.md

```diff
@@ -229,6 +236,106 @@ To use the metrics you can either:
 
 - Scrape the metrics using RDI Prometheus exporter
 
+### Read-through jobs
+
+Read-through jobs are a mandatory part of the read-through pipeline configuration.
+Under the `jobs` directory (parallel to `config.yaml`) you should have a job definition in a YAML file per every key pattern you want to read into cache from a downstream database table in case of cache miss.
+
+The YAML file can be named using the database table name or another naming convention, but has to have a unique name.
+
+Job definition has the following structure:
+
+```yaml
+source:
+  retries: 1
+  timeout: 3 #seconds
+  keyspace:
+    trigger: read-through
+    pattern: emp:*
+  keys:
+    delimiter: ":"
+    # alternatively use expression with RegExp
+    fields:
+    - first_name: 1
+    - last_name: 2
+  connection: my-oracle
+  db: test
+  schema: test
+
+  # One of these should be provided:
+  # - table with columns
+  # - sql
+
+  #table: EMP
+  #columns:
+  # - first_name
+  # - last_name
+  # - birth_date
+
+  # bind variables are marked with a colon and must be defined in the `keys` property.
+  sql: |
+    SELECT emp.*, address.*, kids.*
+    FROM emp
+    LEFT JOIN address ON emp.employee_id = address.employee_id
+    LEFT JOIN kids ON emp.employee_id = kids.employee_id
+    WHERE emp.employee_fname= :first_name AND emp.employee_lname = :last_name;
+
+transform:
+  - uses: rename_field
+    with:
+      fields:
+        - from_field: employee_fname
+          to_field: first_name
+        - from_field: employee_lname
+          to_field: last_name
+output:
+  expire: 100
+```
+
+#### The keyspace section
+
+This section describes the Redis keyspace pattern and job type:
+
+- The `pattern` attribute that would trigger the read-through job.
+- The `trigger` attribute: indicates it is a read-through and not a write-behind job, using the `read-through` value.
+
+#### The keys section
+
+This section has the following attributes:
+
+- The `delimiter` attribute: specifies a delimiter for tokenizing the Redis key. These tokens are used with the `fields` attribute (see below).
+- Alternatively, specify the `expression` attribute. In this case, the regular expression will be used to tokenize the the Redis key. for example, `expression: "(.+):(\\d+)"` will tokenize the key to anything before the colon and a number after the colon.
+- The `fields` attribute is a map of field name and its offset in the array of tokens generated by the `expression` or or the `delimiter` attributes.
+
+#### downstream database information
+
+The `source` section must specify the following attributes:
+
+- `connection` - a connection alias, referring to a connection section in the `config.yaml` file.
+- `db` - the name of the database endpoint to connect use.
```

Comment:
no need, remove, it's part of the connection.

Reply:
done